src/components/taskListContainer/TaskListContainer.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from "react";
+import { Task } from "../../types/Task.types";
+import { getTasks } from "../../utility/api";
+import { NewTaskInputForm } from "../newTaskInputForm/NewTaskInputForm";
+import { TaskItem } from "../taskItem/TaskItem";
+import styles from "./TaskListContainer.module.css";
+
+export const TaskListContainer = (): JSX.Element => {
+  const [tasks, setTasks] = useState<Task[]>([]);
+  const taskItems = Array.from(tasks);
+
+  useEffect(() => {
+    const fetchTasks = async (): Promise<void> => {
+      const data = await getTasks();
+      setTasks(data);
+    };
+    fetchTasks();
+  }, []);
+
+  const taskList = taskItems?.map(
```

Comment:
You can remove this and write it directly inside the task list, see my next comment